aurci/update.py

```diff
@@ -61,6 +60,16 @@ def update_pkgbuild(self):
 
         sha256 = subprocess.run(['sha256sum', fname], check=True, capture_output=True)
         new_sha = "sha256sums=('{}'".format(sha256.stdout.decode('utf-8').split(' ')[0])
+
+        # TODO:
+        #   - Error checking
+        #   - Probably use with ...: notation
+        #   - Check, whether main_dir is only one string (should be)
+        archive = tarfile.open(fname, "r:gz")
+        main_dir = os.path.commonprefix(archive.getnames())
+        new_dir = '_dir="{}"'.format(main_dir)
```

Comment:
This doesn't work in (source of course) repos like `ros_comm`, as the packages are internally grouped into several folders. Additionally, `os.path.commonprefix()` is generally bad, because it returns the string prefix and I guess it's a bug in here, too. I guess `os.path.commonpath()` would be the correct one.

Reply:
This is exactly why I proposed using the release repositories instead.

And yes, `commonpath` would be the right function to use.